includes/class-wcmp-frontend.php

```diff
@@ -467,10 +467,10 @@ private function get_checkout_config() {
                 "pricePickupExpress" => $this->frontend_settings->get_price('pickup_express'),
 
                 "allowMondayDelivery" => $this->frontend_settings->is_enabled('saturday_cutoff'),
-                "allowMorningDelivery" => $this->frontend_settings->is_enabled('morning'),
-                "allowEveningDelivery" => $this->frontend_settings->is_enabled('evening'),
-                "allowSignature" => $this->frontend_settings->is_enabled('signature'),
-                "allowOnlyRecipient" => $this->frontend_settings->is_enabled('only_recipient'),
+                "allowMorningDelivery" => isset(WooCommerce_MyParcel()->export_defaults['age_check']) ? 0 : $this->frontend_settings->is_enabled('morning'),
```

Comment:
We meoten voor wc 5.4 ondersteunen. Ik denk niet dat shorthand if ternary operators mag in 5.4.